### DISCUSSION TITLE
Print templates for orders don't work

### DISCUSSION THREAD

**Discussion by corygibbons** [swellstores/community#185] [Help] [answered]
Creating a custom print template for orders in `Settings → Orders → Print templates` works as expected, however when trying to use the template (`Actions → Print order`) it doesn't work.

What happens is the correct template flashes for a second then reverts back to the default. Selecting any other templates at the point doesn't switch the template either.

**Comment by steven-k1** [comment]
Hey @corygibbons 

Thanks for providing the details. We are aware of this issue, and it is actively being worked on. We'll let you know more as updates become available.

**Comment by swellmike** [comment, marked as answer]
A fix has been deployed for this bug.

**Reply by corygibbons** [reply]
Awesome. Just gave this a test and it works a lot better and can actually use a custom template now so thanks for that.

A heads up that there does still seem to be a small issue where if you have a custom template as the default it sometimes switches to the wrong one (platform default) when opening the modal. Selecting the custom one again in the dropdown makes it show up as expected.

**Reply by ericingram** [reply]
Appreciate the feedback @corygibbons!

**Reply by steven-k1** [reply]
@corygibbons Thanks for the details. Checked on our end as well and can confirm that this is occurring. We will have this logged and let you know more as updates become available.

**Reply by steven-k1** [reply]
@corygibbons The additional issue with the order templates reverting back to the default should now be fixed.

**Reply by corygibbons** [reply]
@StevenSwell Just gave it a try and seems to be working perfectly, thanks!
